### DISCUSSION TITLE
Can you run the medium model on an Intel Macbook Pro from 2018?

### DISCUSSION THREAD

**Discussion by codingbutstillalive** [ggml-org/whisper.cpp#933] [Q&A]
I assume the large model will totally depend on a CUDA GPU. How about the medium one, though? Any experiences?

**Comment by kevin2379** [comment]
I've been doing a lot of testing while building AirCaption on a 2018 15" MacBook Pro. It runs both the medium and large models just fine. Transcribing a 10 min video took about 12 min with the medium model and 20-25 min with the large model. A 13" MacBook Pro might take a bit longer.

**Reply by codingbutstillalive** [reply]
Thank you very, very much!
